sycl/unittests/assert/assert.cpp

```diff
@@ -256,153 +243,6 @@ static void setupMock(sycl::unittest::PiMock &Mock) {
   Mock.redefineBefore<PiApiKind::piEventsWait>(redefinedEventsWaitPositive);
 }
 
-namespace TestInteropKernel {
-const sycl::context *Context = nullptr;
-const sycl::device *Device = nullptr;
-int KernelLaunchCounter = ::KernelLaunchCounterBase;
-
-static pi_result redefinedKernelGetInfo(pi_kernel Kernel,
-                                        pi_kernel_info ParamName,
-                                        size_t ParamValueSize, void *ParamValue,
-                                        size_t *ParamValueSizeRet) {
-  if (PI_KERNEL_INFO_CONTEXT == ParamName) {
-    pi_context PiContext =
-        sycl::detail::getSyclObjImpl(*Context)->getHandleRef();
-
-    if (ParamValue)
-      memcpy(ParamValue, &PiContext, sizeof(PiContext));
-    if (ParamValueSizeRet)
-      *ParamValueSizeRet = sizeof(PiContext);
-
-    return PI_SUCCESS;
-  }
-
-  if (PI_KERNEL_INFO_PROGRAM == ParamName) {
-    pi_program PIProgram = nullptr;
-    pi_result Res = mock_piProgramCreate(/*pi_context=*/0x0, /**il*/ nullptr,
-                                         /*length=*/0, &PIProgram);
-    EXPECT_TRUE(PI_SUCCESS == Res);
-
-    if (ParamValue)
-      memcpy(ParamValue, &PIProgram, sizeof(PIProgram));
-    if (ParamValueSizeRet)
-      *ParamValueSizeRet = sizeof(PIProgram);
-
-    return PI_SUCCESS;
-  }
-
-  if (PI_KERNEL_INFO_FUNCTION_NAME == ParamName) {
-    static const char FName[] = "TestFnName";
-    if (ParamValue) {
-      size_t L = strlen(FName) + 1;
-      if (L < ParamValueSize)
-        L = ParamValueSize;
-
-      memcpy(ParamValue, FName, L);
-    }
-    if (ParamValueSizeRet)
-      *ParamValueSizeRet = strlen(FName) + 1;
-
-    return PI_SUCCESS;
-  }
-
-  return PI_ERROR_UNKNOWN;
-}
```

Comment:
After the changes to sycl-rt opencl interop apis, it now calls cl entry-points, so these tests were trying to make a mock cl handles and if we called sycl with this fake handles, it will seg-fault from fake cl handles passed. So, the other options was to make a non fake CL handles which will make a code exactly like in this kind of interop e2e [tests](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/DeprecatedFeatures/kernel_interop.cpp), So, I was wandering if these kind of tests are really adding value here on e2e tests, or is there a way to test this without having to do the whole initialization pipeline, like e2e tests does to test interop functionality.